src/Header.spec.js

```diff
@@ -0,0 +1,11 @@
+import { render, screen } from '@testing-library/react'
+import Header from './Header'
+
+describe('render Header', () => {
+  it('shows Header Image', () => {
+    render(<Header />)
+
+    const header = screen.getByRole('img')
+    expect(header).toBeInTheDocument()
```

Comment:
```suggestion
    const logo = screen.getByRole('img')
    expect(logo).toBeInTheDocument()
```
This is not the header